### DISCUSSION TITLE
[Feature request] Adjustable table column sizes.

### DISCUSSION THREAD

**Discussion by XikoCat** [arnoudkooi/SN-Utils#368] [Ideas]
Here is something that I wish SN Allowed me to do:

Allow custom width for columns when on the table view.

This could be achieved if we could set relative dimensions for every column for each view on all tables that we want to customize.
Ideally this would persist and resize the table every time it is loaded.
Given SN uses Bootstrap as a base for CSS Layout, setting values that total in 12 could work. Values over 12 might require the user to scroll right to see more.

Another way, more temporary, could be to just let any user drag the column border to resize it. A similar behavior as an Excel spreadsheet.
Meaning if I just wanted to see the rest of the values that are cropped I could just resize the column.
This would reset every time the page is reloaded and default back to the previous suggestion.

I think it could improve the overall  productivity to have this quality of life improvement.

**Comment by arnoudkooi** [comment]
The text in the table cell is actually cut off and not truncated via CSS, so I don't think dragging the columns will help...
